ddtrace/debugging/_async.py

```diff
@@ -3,19 +3,23 @@
 from typing import List
 
 from ddtrace.debugging._snapshot.collector import SnapshotContext
+from ddtrace.internal import compat
 
 
 async def dd_coroutine_wrapper(coro, contexts):
     # type: (CoroutineType, List[SnapshotContext]) -> CoroutineType
+    start_time = compat.monotonic_ns()
     try:
         retval = await coro
+        end_time = compat.monotonic_ns()
```

Comment:
would it make sense to put this in a `finally` block? Or should we not because of timing?

Reply:
I think this way we are more in control of when `compat.monotonic_ns` is called.